_sass/module/_brand.scss

```diff
@@ -1,5 +1,6 @@
 .brand {
   font-family: $font-family-sans-serif;
+  color: $color-white;
```

Comment:
This change is probably too broad, since it will apply to any `brand` component on the site. We probably want to make a rule thats specific to the landing page. Honestly, the contrast of the blue brand color on the background isn't large enough, so I think we want the entire text to be white, like it is on the desktop version, and not use `brand` at all.